src/main/java/org/payw/npoem/domain/user/CustomUserDetailService.java

```diff
@@ -0,0 +1,21 @@
+package org.payw.npoem.domain.user;
+
+import org.payw.npoem.domain.user.UserRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import org.springframework.stereotype.Service;
+
+@RequiredArgsConstructor
+@Service
+public class CustomUserDetailService implements UserDetailsService {
```

Comment:
[fix]
i think service layer should be located service package
move loadUserByusername method to UserService

Reply:
Ok, I will.